index.js

```diff
@@ -94,15 +99,49 @@ const getAddresses = () => {
   return _rlAddressToRequests
 }
 
+
+const hash = (address) => {
+  return Crypto.SHA256(address).toString(Crypto.enc.Hex);
+
+}
+
 const setTotalActiveUsers = (req) => {
-  const address = req.connection.remoteAddress
-  const lastRequestAt = _rlAddressToRequests[address] ? _rlAddressToRequests[address].lastRequestAt : null
+  const address = hash(req.connection.remoteAddress)
+  incrementActiveUsers(address)
+}
 
-  if(!lastRequestAt || lastRequestAt.diff(_lastActiveTimeout) < 0){ //The last connection was before we zeroed the totalActiveUsers field
+incrementActiveUsers = (address) => {
+  if(!userActive(address)){ //The last connection was before we zeroed the totalActiveUsers field
     _totalActiveUsers++
   }
 }
 
+decrementActiveUsers = (address) => {
+  if (userActive(address)) { //The last connection was after we zeroed the totalActiveUsers field
+    _totalActiveUsers--
+  }
+}
+
+const userActive = (address) => {  
+  const lastRequestAt = _rlAddressToRequests[address] ? _rlAddressToRequests[address].lastRequestAt : null
```

Comment:
This can be simplified to `const lastRequestAt = _rlAddressToRequests[address] && _rlAddressToRequests[address].lastRequestAt`

Reply:
Ok